docs/schemas/Cell-Annotation-and-Taxonomy/Cell-Annotation-Schema.csv

```diff
@@ -0,0 +1,14 @@
+matrix_file_id,5d625688-96da-4c65-97b9-211cbcad4aea,,,"A resolvable ID for a cell by gene matrix file in the form namespace:accession, e.g. CellXGene_dataset:8e10f1c4-8e98-41e5-b65f-8cd89a887122. Please see https://github.com/cellannotation/cell-annotation-schema/registry/registry.json for supported namespaces.",,,string,CellXGene_dataset:8e10f1c4-8e98-41e5-b65f-8cd89a887122,,,,,,
+title,5aab17df-0830-44d4-bcd8-f954695867d0,,,The title of the dataset. This MUST be less than or equal to 200 characters. e.g. 'Human retina cell atlas - retinal ganglion cells'.,FALSE,,string,'Human retina cell atlas - retinal ganglion cells',,,,,,
+description,1b5ff66c-0e00-4a29-8ccf-65dbf59d79da,,,"The description of the dataset. e.g. 'A total of 15 retinal ganglion cell clusters were identified from over 99K retinal ganglion cell nuclei in the current atlas. Utilizing previous characterized markers from macaque, 5 clusters can be annotated.'.",,,string,"'A total of 15 retinal ganglion cell clusters were identified from over 99K retinal ganglion cell nuclei in the current atlas. Utilizing previous characterized markers from macaque, 5 clusters can be annotated.'",,,,,,
+cellannotation_schema_version,195cbdbf-d486-4d54-9c1d-83edf0a44ec5,,,"The schema version, the cell annotation open standard. Current version MUST follow 0.1.0This versioning MUST follow the format '[MAJOR].[MINOR].[PATCH]' as defined by Semantic Versioning 2.0.0, https://semver.org/.",,,string,1.2.1,,,,,,
+cellannotation_timestamp,37da06ba-9c15-405f-b5d6-f8d2bf5fc3a3,,,The timestamp of all cell annotations published (per dataset). This MUST be a string in the format '%yyyy-%mm-%dd %hh:%mm:%ss'.,,,date-time,,,,,,,
+cellannotation_version,439f52e8-e2e6-406c-850c-434d448c8b6d,,,"The version for all cell annotations published (per dataset). This MUST be a string. The recommended versioning format is '[MAJOR].[MINOR].[PATCH]' as defined by Semantic Versioning 2.0.0, https://semver.org/.",,,string,1.2.1,,,,,,
+cellannotation_url,e47029df-a5cf-49d0-b8de-76302a1e6fbb,,,A persistent URL of all cell annotations published (per dataset).,,,string,,,,,,,
+author_list,15e4be61-b1fb-49a5-9a81-7fed61138256,,,"This field stores a list of users who are included in the project as collaborators, regardless of their specific role. An example list; '['John Smith', 'Cody Miller', 'Sarah Jones']'.",,,string," '['John Smith', 'Cody Miller', 'Sarah Jones']'",,,,,,
+author_name,0ea83cdb-cd06-4ef8-84c1-6aec29220759,,,Primary author's name. This MUST be a string in the format [FIRST NAME] [LAST NAME].,FALSE,,string,,,,,,,
+author_contact,fa084043-3c8e-47cb-959e-2f3daf2165ec,,,Primary author's contact. This MUST be a valid email address of the author.,,,"string, format: email",,,,,,,
+orcid,2c96776b-b158-4e0c-ba67-ea7a63efb1e8,,,Primary author's orcid. This MUST be a valid ORCID for the author.,,,string,,,,,,,
+labelsets,7fa0b7a1-6af7-4f79-8da0-29d7950ea736,,,see labelsets table,FALSE,,array,,,,,,,

```

Comment:
I understand that the range of this slot/attribute should be a class `Lableset` and  could be multivalued, is that right? I guess description still can be useful.

similar comment to the next line